kedro-org/kedro-viz
# Refactor Flowchart models from dataclass to pydantic base models

## Description

Partially Resolves #1538 

## Development notes

* Replaced all data classes from models/flowchart.py with pydantic models
* Used root-validators and field validators
* Modified pytests

## QA notes

* All features in Kedro Viz should run normally. 
* Start the app either using `kedro viz` command from demo-project or run the dev server using `make run`
* Run pytests using `make pytest`

## Checklist
